spec/offchain.tex

```diff
@@ -185,45 +189,75 @@ \subsubsection{Processing transactions off-chain}
 receiving party $\party_{i}$ is the next snapshot
 leader, a message to request snapshot signatures $\hpRS$ is sent. \\
 
-\dparagraph{$\hpRD$.}\quad Upon receiving request $(\hpRD,\tx)$, the transaction is
-checked against the \emph{local} ledger state. If decommit is not applicable yet
-or another decommit is still pending, the protocol does $\KwWait$ to retry later or
-eventually marks the decommit as invalid. In case there is no snapshot in flight
-and party is the leader then a snapshot request $\hpRS$ is
-sent containing the decommit transaction $\tx_{\omega}$. \\
+\dparagraph{$\hpRD$.}\quad Upon receiving request $(\hpRD,\tx_\omega)$, \red{if
+  another decrement (or increment) is pending to be settled the transaction is
+  rejected. Otherwise} the transaction is checked against the \emph{local}
+  ledger state and if it is not applicable yet or another decommit is still, the protocol does $\KwWait$ to
+  retry later or eventually marks the decommit as invalid. \red{After applying $\tx$, it's inputs
+  are removed from \emph{local} ledger state so that they are not available any
+  more and the outputs are kept in the local state ($U_\omega$) to be posted
+  on-chain in the next snapshot.} Finally, if there is no current snapshot in flight
+  ($\hats = \bar{\mc S}.s$) and the receiving party $\party_{i}$ is the next
+  snapshot leader, a message to request snapshot signatures $\hpRS$ containing
+  the decrement transaction $\tx_\omega$ is sent. \\
 
 \dparagraph{$\mathtt{decrementTx}$.}\quad Upon observing the \mtxDecrement{}
-transaction, which removed outputs $U_{\omega}$ from the head, the corresponding
-decommit transaction is cleared and the observed version $v$ is used for future
-snapshots by
-setting $\hatv \gets v$. Note that the version of the open head state is incremented on each \mtxDecrement{} transaction as described in Section~\ref{sec:decrement-tx} \\
+transaction, which removed outputs $U$ from the head, the corresponding
+pending decrement transaction is cleared and the observed version $v$ is used for future
+snapshots by setting $\hatv \gets v$.  Note that the version of the open head state
+is incremented on each \mtxDecrement{} transaction as described in Section~\ref{sec:decrement-tx}
+\todo{missing to mention the multicast of $\hpRS$} \\
+
+\red{\dparagraph{$\hpRI$.}\quad Upon receiving request $(\hpRI,\tx)$, if
+  another decrement (or increment) is pending to be settled the transaction is
+  rejected. Otherwise the transaction inputs are kept in the local state ($U_\alpha$) to be included
```

Comment:
It is not strictly "rejected", but we wait for it to not be the case. Let's use wording similar to the `ReqTx` and `RecDec`:

"If another commit or decommit is pending, the protocol does wait to retry later or eventually marks the decommit as invalid."